src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -183,7 +184,45 @@ def get(self, request, organization):
 
         results = list(cursor_result)
 
-        context = serialize(results, request.user, serializer())
+        skip_snuba_fields = {
+            "query",
+            "status",
+            "bookmarked_by",
+            "assigned_to",
+            "unassigned",
+            "subscribed_by",
+            "active_at",
+            "first_release",
+            "first_seen",
+            "message",  # not 100% on this one
+        }
+        snuba_filters = []
+        if query_kwargs["search_filters"] is not None:
+            snuba_filters = [
+                convert_search_filter_to_snuba_query(search_filter)
+                for search_filter in query_kwargs["search_filters"]
+                if search_filter.key.name not in skip_snuba_fields
+            ]
+
+        lifetime_stats = serialize(results, request.user, serializer())
+
+        if features.has("organizations:dynamic-issue-counts", organization, actor=request.user):
+            context = serialize(results, request.user, serializer(start=start, end=end))
```

Comment:
isn't context supposed to be a subset of the result of lifetime_stats ? It seems the only difference is start and end date. We should be able to generate context of line 210 from the result of line 207 without doing another Snuba query.

Reply:
@fpacifici What's the best way to do this? Something like
`count(if(subset_start <= ts < subset_end , 1, null)) as subset_count`
`count(distinct if(subset_start <= ts < subset_end , user, null)) as subset_users`
?

Are we able to do this via snuba at the moment?

Reply:
@fpacifici Just bumping about this, is the above how we'd achieve this?

Reply:
Can we merge this since it is behind a feature switch and I will modify these queries before general release?

Reply:
I meant doing one query only to get lifetime_stats as of today and then extract `context` from the result of lifetime_stats, to my understanding context should be a subset of what was returned to populate lifetime_stats. 

But feel free to go ahead and merge for internal testing as long as we reliably keep track of the issue.